.golangci.yml

```diff
@@ -13,3 +13,7 @@ linters:
     - nilerr
     - usestdlibvars
     - wastedassign
+issues:
+  exclude-files:
+    - '(.+)_test\.go'
+    - cmd/operator/internal/controller/kubearchiveconfig_controller.go
```

Comment:
Could you add some comment to reflect WHY these files are being excluded?

Reply:
my bad. Added explainations